pkg/sonar/sonar_test.go

```diff
@@ -25,14 +26,14 @@ func TestReadTaskReport(t *testing.T) {
 		result, err := ReadTaskReport("./testData/missing")
 		// assert
 		assert.Empty(t, result.ProjectKey)
-		assert.EqualError(t, err, "open testData/missing/.scannerwork/report-task.txt: no such file or directory")
+		assert.Regexp(t, "open testData.missing..scannerwork.report-task.txt: (no such file or directory|The system cannot find the path specified.)", err)
```

Comment:
Why is `fromSlash` not used here?

Reply:
the backslashes of FromSlash() had to be escaped else the regular expression is wrong. It is simplier to use a dot instead.

Reply:
instead:

`assert.True(t, filepath.FromSlash("open testData/missing/.scannerwork/report-task.txt: The system cannot find the path specified.") == err.Error() || filepath.FromSlash("open testData/missing/.scannerwork/report-task.txt: no such file or directory") == err.Error())
`

Reply:
ah, now I noticed the windows specific message, missed that before: `(no such file or directory|The system cannot find the path specified.)`

Reply:
I'd prefer the simple non-regex solution.:

`assert.True(t, ... || ...)`